doc/userguide/devguide/codebase/contributing/github-pr-workflow.rst

```diff
@@ -0,0 +1,49 @@
+GitHub Pull Request Workflow
+============================
+
+Draft Pull Requests
+~~~~~~~~~~~~~~~~~~~
+
+A Pull Request (PR) should be marked as `draft` if it is not intended to be merged as is,
+but is waiting for some sort of feedback.
+The author of the PR should explicit with what kind of feedback is expected
```

Comment:
I let this one slip in the last review >_<'
```suggestion
The author of the PR should be explicit with what kind of feedback is expected
```

Reply:
ok